package.json

```diff
@@ -100,7 +100,7 @@
     "copy:types": "grunt copy:types",
     "build:types": "yarn clean:types && yarn generate:types && yarn copy:types",
     "test:parity-setup": "mkdir -p test-reports/testcafe && yarn generate-config && yarn workspace v3 dev",
-    "test:parity-run": "OKTA_SIW_GEN3=true yarn testcafe -c 2 chrome:headless --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
+    "test:parity-run": "OKTA_SIW_GEN3=true yarn testcafe -c 2 \"chrome:headless --force-device-scale-factor=1\" --reporter spec,xunit:build2/reports/junit/testcafe-results.xml",
```

Comment:
must set `--force-device-scale-factor=1` in chrome browser to keep screenshots quality consistent between different computers